### ISSUE TITLE
[Enhancement] Automatic execution of LongTapCommand

### ISSUE THREAD

**Issue by SimonSchnitker** [mrxten/XamEffects#46]
Hello,

If I use the LongTap command, it will only be executed if it is held down for some time and then released.

As a user, I would find it better if I only have to hold down for a while and then the command is executed automatically.

What do you mean?

**Comment by mrxten** [comment]
Hi, if I correct remember, long tap command executes after some time, not after releasing.

**Comment by SimonSchnitker** [comment]
No, unfortunately not. The LongTap command is only triggered when an element is released.
Please try it out.

**Comment by mrxten** [comment]
Which platform and xamarin forms version?26 янв. 2020 г. 14:32 пользователь SymxnCreator <notifications@github.com> написал:No, unfortunately not. The LongTap command is only triggered when an element is released.
Please try it out.

**Comment by SimonSchnitker** [comment]
Xamarin.Forms on Android
Xamarin.Forms version: 4.4.0

I tested it on an Android 7 and Android 10 device. In both cases, the LongTap command is only triggered when you release it.
Unfortunately, I cannot test whether the problem also exists on iOS at the moment.

**Comment by diegoserpa** [comment]
I experienced the same issue.
In my case, I have a view wich I attach two commands:
- Tap: to navigate to another page;
- LongTap: to display an action sheet;

When I tap the view, it navigates to another page as expected. If I long press the view, it navigates to the page (behavior wich is expected in a single short tap) and when I release the action sheet is displayed (the behavior expectd in a long tap).

Additional info:
Xamarin.Forms on Android
Xamarin.Forms version: 4.3.0.991221
